Add blog post on 2021 matching campaign (#1512)
writethedocs/www: docs/blog/2021-matching-campaign.rst

.. post:: Apr 26, 2021
   :tags: bipoc, statement

Announcing our 2021 donation matching campaign
==============================================

> We're proud to announce the second round of our $1,500 matching campaign to support social justice in our communities. `Last year <https://www.writethedocs.org/blog/we-stand-with-the-black-community/>`_ we directed donations to the Black community, and this year we're expanding that aim to focus on the wider BIPOC community. We want it to be clear that we support and care about the members of our community, and to show this through our words and our actions as an organization.

Reviewer: We say "community" a lot.

> We want to continue to make a difference in our local communities, and also encourage members of the Write the Docs community to engage with local organizations in their region. Based on our Portland conference, we're suggesting three Oregon charities we believe are doing important work. That said, you can also donate to a charity local to you doing similar outreach in your community.

Reviewer: ```suggestion
We want to continue to make a difference in our local communities, and also encourage members of the Write the Docs community to engage with local organizations in their region. Based on our Portland conference, we're suggesting three Oregon charities we believe are doing important work. That said, you can also donate to a charity local to you doing similar outreach.
```

* `Water for Warm Springs <https://mrgfoundation.org/the-chuush-fund-water-for-warm-springs/>`_ - A campaign to ensure clear drinking water, a basic human right, for the Confederated Tribes of Warm Springs
* `Asian Pacific American Network of Oregon <https://www.apano.org/>`_ - A group that unites Asian and Pacific Islanders to achieve social justice

We will match donations to these organizations and others that are supporting these efforts, up to a total of $1,500. Please `forward receipts <mailto:support@writethedocs.org>`_ to us with a request for a matching donation. In the next couple months, we will post another blog post with the results of these donations, similar to `last year <https://www.writethedocs.org/blog/donations-black-community/>`_.

We'd also like to thank all those in our community who are doing the work to make BIPOC members of the community feel welcome and included. This includes a BIPOC unconference at our Portland conference, a `#bipoc` channel on our Slack network, and the `links shared <https://www.writethedocs.org/blog/newsletter-april-2021/#what-we-re-reading-listening-to-watching>`_ recently in our newsletter.

> We hope that we can do our small part to move toward a more just world, and we appreciate everyone in the community who works with us towards these goals.

Reviewer: ```suggestion
We hope that these actions move us all towards a more just world, and we appreciate everyone in the community who works with us towards these goals.
```
